src/Report/JunitReporter.php

```diff
@@ -34,6 +34,10 @@ public function getFormat()
      */
     public function generate(ReportSummary $reportSummary)
     {
+        if (!extension_loaded('dom')) {
```

Comment:
let us keep this PR as a quickfix, adding nice ahead-of time warning would be great improvement, but it shall deserve a dedicated PR to provide it in nice version (instead of version provided fast)